### ISSUE TITLE
I need help making this strategy

### ISSUE THREAD

**Issue by Arionfox** [binary-com/binary-bot#2035]
when the last two digits (after dot) are at x.00 open a trade for rise

![www binary](https://user-images.githubusercontent.com/53241830/67016888-4cdd9f80-f0cf-11e9-9466-e0a569c758e0.png)

Thank you very much in advance

**Comment by sinung21** [comment]
[00- rise.zip](https://github.com/binary-com/binary-bot/files/3739842/00-.rise.zip)

**Comment by ceakuk** [comment]
@Arionfox 
probability of last digit being 0 is 1 out of 10
probability of  last and 2nd last being 0 is 1 out of 100
that means you get  your signal 1 every 100 ticks on average. maybe even 1 out of 1000 ticks.

the bot will just lay idle the whole day plus  what is the logic. these are random numbers anyway. one digit does not affect the other. they are independent

**Comment by Richard610** [comment]
HELLO FRIENDS GOOD NIGHT. A QUESTION HOW CAN I ASK FOR THIS BOT TO FOLLOW NORMAL IS TO SAY IF I WIN OR LOSE THAT I STILL RUNNING AS IF I WIN I WAS LOOKING BENEFIT AND I HAVE TO PLACE YOU STOP AND RETURN TO START THEM THANK YOU
![Imagen2](https://user-images.githubusercontent.com/56711262/67069229-b4055d80-f17c-11e9-98e4-ae1d026e0dc0.png)

**Comment by navneetsinh** [comment]
> HELLO FRIENDS GOOD NIGHT. A QUESTION HOW CAN I ASK FOR THIS BOT TO FOLLOW NORMAL IS TO SAY IF I WIN OR LOSE THAT I STILL RUNNING AS IF I WIN I WAS LOOKING BENEFIT AND I HAVE TO PLACE YOU STOP AND RETURN TO START THEM THANK YOU
> ![Imagen2](https://user-images.githubusercontent.com/56711262/67069229-b4055d80-f17c-11e9-98e4-ae1d026e0dc0.png)

you can use prompt message after win/loss result.

**Comment by Richard610** [comment]
HOLA AMIGO. DISCULPA MI INGLES ESTOY USANDO UN TRADUCTOR .. TE QUERIA
MOLESTAR COMO SE LE PUEDE COLOCAR INTERES COMPUESTO A 3 O 5 NIVELES A ESTE
BOT .



HELLO FRIEND. I apologize for my English. I am using a translator. I
would like to bother you as you can place compound interest at 3 or 5
levels at this boot. THANK YOU FOR YOUR FRIEND ATTENTION.

**Comment by Richard610** [comment]
Hola amigo te queria molestar.
soy nuevo con esto de los bots para binary.
pero estoy interesado en que e ayudes hacer uno.
la idea es que aga una operacion a cada minuto.
pero al segundo 33 ingrese la operacion siempre a ese segundo.
es decir Hora/minutos/segundos 33 simpre que ingrese una operacion en ese
segundo crees que se pueda hacer
te agradeceria bastante amigo. exitos


Hello friend, I wanted to bother you. I'm new with this one for bots for
binary, but I'm interested in helping you make one. the idea is to add an
operation every minute, but at the second 33, always enter the operation at
that second, that is to say, hour / minutes / seconds, 33 if you always
enter an operation in that second, you think it can be done, I would
appreciate it, my friend

**Comment by navneetsinh** [comment]
@Richard610 
Try this,
![Opera Snapshot_2019-11-05_143849_bot binary com](https://user-images.githubusercontent.com/42480287/68193929-11513980-ffda-11e9-9cca-5caf7b27f237.png)

**Comment by Richard610** [comment]
hola amigo disculpa que te moleste demaciado.

te queria molestar este bot lo encontre gratis pero lo e modificado un poco
y te queria molestar si me ayuda a hacer interés compuesto a 3 o 5 niveles.
es decir si gana que sume las ganancias. muchas gracias por tu atencion.
Bendiciones amigo


Hi friend, I'm sorry to bother you too much. I wanted to bother you this
bot I found it for free but I modified it a bit and I wanted to bother you
if it helps me to make compound interest at 3 or 5 levels. that is, if you
win that you add the profits. thank you very much for your attention.
Blessings friend
